### ISSUE TITLE
ios app sim building error

### ISSUE THREAD

**Issue by Saleciani** [LtbLightning/bdk-flutter#133]
when trying to run a ios app, on ^0.31.2-dev.1

the build fails with

```
Launching lib/main.dart on iPhone 15 Pro Max in debug mode...
Running pod install...
Running Xcode build...
Xcode build done.                                            6.6s
Failed to build iOS app
Error (Xcode): Multiple commands produce '/Users/joaolobo/code/project/satsails/build/ios/Debug-iphonesimulator/bdk_flutter/libbdk_flutter.a'

Could not build the application for the simulator.
Error launching application on iPhone 15 Pro Max.

```

**Comment by BitcoinZavior** [comment]
@JoaoLobo94 Do you mean you are facing this issue only with v0.31.2-dev.1?
I will try and take a look tomorrow or on Monday.
I believe I have used it in a brand-new Flutter app and it worked fine.

**Comment by JoaoLobo94** [comment]
Yes it only happens on this version. I downgraded to 0.30 and it compiled fine.

I tried to flutter clean and that did not seem to make the trick.

**Comment by JoaoLobo94** [comment]
Hello, is there any update on this bug? had a look and i am still experiencing it :)

**Comment by kumulynja** [comment]
> Hello, is there any update on this bug? had a look and i am still experiencing it :)

Hi @JoaoLobo94 , I tried it out on an iPhone 15 Pro Max simulator too and it worked. I tried for iOS version 17.0 and 17.5, both worked. So could you try out following things from your side, please:

- Try with bdk_flutter v0.31.2-dev.**2**, this is the latest version.
- Check if you have the latest version of Xcode and iOS for your simulators. If not, please share the versions you were using so I can test with those.
- Try the answers here: https://stackoverflow.com/questions/52398521/flutter-ios-build-failure-error-with-multiple-commands-after-the-xcode-upgrade

Also feel free to join the Let there be Lightning's Discord server here to ask any questions or support: https://discord.gg/6BhQm6UH .
